mocha.entry.js

```
@@ -83,9 +87,9 @@ describe('[ModernizrWebpackPlugin] Build Tests', function () {
     webpackConfig.output.publicPath = 'public/';
     webpack(webpackConfig).then(function () {
       fs.readFileAsync(path.resolve(OUTPUT_PATH, 'index.html'), 'utf8').then(function (data) {
-        expect(/<script\ssrc="public\/modernizr-bundle.js">/.test(data)).to.be.true;
+        expect(/<script(.*)src="public\/entry-bundle.js">/.test(data)).to.be.true;
         done();
-      })
+      }).catch(function (error) { });
```

Why are we swallowing errors in a test? Especially given there's a `.catch()` below that will trap them and pass them on to the test runner.

---

That's a good question, I'll remove it.